lib/releaselib.sh

```diff
@@ -33,9 +33,10 @@ readonly GCRIO_PATH_PROD="k8s.gcr.io"
 readonly GCRIO_PATH_PROD_GEO_ASIA="asia.gcr.io/k8s-artifacts-prod"
 readonly GCRIO_PATH_PROD_GEO_EU="eu.gcr.io/k8s-artifacts-prod"
 readonly GCRIO_PATH_PROD_GEO_US="us.gcr.io/k8s-artifacts-prod"
-readonly GCRIO_PATH_TEST="gcr.io/k8s-staging-kubernetes"
+readonly GCRIO_PATH_STAGING="gcr.io/k8s-staging-kubernetes"
+readonly GCRIO_PATH_MOCK="${GCRIO_PATH_STAGING}/mock"
 
-readonly KUBE_CROSS_REGISTRY="us.gcr.io/k8s-artifacts-prod/build-image"
+readonly KUBE_CROSS_REGISTRY="${GCRIO_PATH_PROD}/build-image"
```

Comment:
Is this missing a part of the URL?

Before it was `us.gcr.io/k8s-artifacts-prod/build-image`, but `GCRIO_PATH_PROD` is set to `k8s.gcr.io`, so this will equal to `k8s.gcr.io/build-image` at the end if I'm not mistaken.

Is this change intended?

Reply:
After thinking a bit more, this change makes sense. It's a bit early for me. :grimacing:

Reply:
No worries! :sweat_smile:
...and yes, `k8s.gcr.io/build-image` is the intended path.